flows/field.go

```diff
@@ -23,23 +23,23 @@ func (f *Field) Asset() assets.Field { return f.Field }
 
 // Value represents a value in each of the field types
 type Value struct {
-	text     types.XText
-	datetime *types.XDateTime
-	number   *types.XNumber
-	state    LocationPath
-	district LocationPath
-	ward     LocationPath
+	Text     types.XText      `json:"text"`
+	Datetime *types.XDateTime `json:"datetime,omitempty"`
+	Number   *types.XNumber   `json:"number,omitempty"`
+	State    LocationPath     `json:"state,omitempty"`
+	District LocationPath     `json:"district,omitempty"`
+	Ward     LocationPath     `json:"ward,omitempty"`
```

Comment:
Yay, had just created my own mailroom type for this since I couldn't make them.

Reply:
yeah we had a `fieldValueEnvelope` previously but then noticed it was the same as `Value`